app/views/controllers/comments/_comments_for_object.erb

```diff
@@ -21,9 +20,9 @@ tag.div(
                         class: "float-right")) if controls
       end
     end,
-    tag.div(id: "comments",
-            class: "list-group list-group-flush comments",
-            data: { controller: "section-update" }) do
+    tag.div(
+      id: "comments", class: "list-group list-group-flush comments"
```

Comment:
Why is the `data: { controller: "section-update" }` not needed in this case?

Reply:
Looking at it more, I think I get it.  This is the container for the comments so no update is needed.  The embedded comments/comment is where the real updates happen.